File: API/models/Inventory.js
'use strict';
const { Model } = require('sequelize');

module.exports = (sequelize, DataTypes) => {
  class Inventory extends Model {
    static associate(models) {
      Inventory.belongsTo(models.Material, {
        foreignKey: 'material_id',
        as: 'material',
      });
    }
  }

  Inventory.init({
    material_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
    },
    location: {
      type: DataTypes.STRING,
    },
    quantity: {
      type: DataTypes.INTEGER,
      defaultValue: 0,
    }
  }, {
    sequelize,
    modelName: 'Inventory',
    tableName: 'inventory',
    underscored: true,
    timestamps: true,
  });

  return Inventory;
};

File: StockFlowAPI/Models/Inventory.cs
namespace StockFlowAPI.Models
{
    public class Inventory
    {
        public int Id { get; set; }
        public int MaterialId { get; set; }
        public string Location { get; set; } = string.Empty;
        public int Quantity { get; set; }

        public Material Material { get; set; } = null!;
    }

}

